manager/integration/tests/test_settings.py

```diff
@@ -782,3 +786,169 @@ def test_setting_concurrent_rebuild_limit():  # NOQA
     (Not sure if we need to put this specific case as a separate test in
      test_engine_upgrade.py)
    """
+
+    # Step 1-1
+    update_setting(client,
+                   "concurrent-replica-rebuild-per-node-limit",
+                   "1")
+
+    # Step 1-2
+    volume1_name = "test-vol-1"  # NOQA
+    volume1 = create_and_check_volume(client, volume1_name, size=str(3 * Gi))
+    volume1.attach(hostId=get_self_host_id())
+    volume1 = wait_for_volume_healthy(client, volume1_name)
+
+    volume2_name = "test-vol-2"  # NOQA
+    volume2 = create_and_check_volume(client, volume2_name, size=str(3 * Gi))
+    volume2.attach(hostId=get_self_host_id())
+    volume2 = wait_for_volume_healthy(client, volume2_name)
+
+    # Step 1-3
+    os.system("dd if=/dev/urandom of=/dev/longhorn/{} bs=1M count=2000".
+              format(volume1_name))
+    os.system("dd if=/dev/urandom of=/dev/longhorn/{} bs=1M count=2000".
+              format(volume2_name))
+
+    # Step 1-4, 1-5
+    delete_replica_on_test_node(client, volume1_name)
+    wait_for_rebuild_start(client, volume1_name)
+    delete_replica_on_test_node(client, volume2_name)
+    wait_for_rebuild_complete(client, volume1_name)
+    wait_for_rebuild_start(client, volume2_name)
+    wait_for_rebuild_complete(client, volume2_name)
+
+    # Step 1-6
+    wait_for_volume_healthy(client, volume1_name)
+    wait_for_volume_healthy(client, volume2_name)
+    delete_replica_on_test_node(client, volume1_name)
+
+    # Step 1-7
+    wait_for_rebuild_start(client, volume1_name)
+    delete_replica_on_test_node(client, volume2_name)
+    update_setting(client,
+                   "concurrent-replica-rebuild-per-node-limit",
+                   "2")
+    wait_for_rebuild_start(client, volume2_name)
+
+    # Step 1-8
+    wait_for_rebuild_complete(client, volume1_name)
+    wait_for_rebuild_complete(client, volume2_name)
+
+    # Step 1-9
+    update_setting(client,
+                   "concurrent-replica-rebuild-per-node-limit",
+                   "1")
+
+    # Step 1-10
+    delete_replica_on_test_node(client, volume1_name)
+    wait_for_rebuild_start(client, volume1_name)
+    crash_replica_processes(client, core_api, volume1_name)
+    delete_replica_on_test_node(client, volume2_name)
```

Comment:
we don't need this

Reply:
I mean with the suggestion https://github.com/longhorn/longhorn-tests/pull/975/files#r888524038, we don't need this step because we already crash the replica of volume 2 earlier. How do you think?

Reply:
I prefer to crash the replicas in sequence so that we know which replica should be rebuilt and which one shouldn't.